### ISSUE TITLE
Cosmetic formatting problems within initial README.md [NOBUG / User Error]

### ISSUE THREAD

**Issue by mcm1957** [AlCalzone/create-adapter#971]
When creating a new adapter README.md is fille with some developer instructions.

A) The following sentence is hard to read as all Backslashes and colons are missing:

1. Add a new secret under _6httpsgithub.comiobroker-community-sadaptersioBroker.nsclientsettingssecrets_. It must be named `AUTO_MERGE_TOKEN` and contain a personal access token with push access to the repository, e.g. yours. You can create a new token under _httpsgithub.comsettingstokens._

The same is true for other links, i.e. 

_(httpsgithub.comioBrokerioBroker.repositories#development-and-coding-best-practices)_ 

_The ioBroker.admin interface will then be available at httplocalhost8181_

B) I suggest to store the developer information within a seperate file, i.e. README-developer.md and  add this file to .npmignore. This way README.md5 could be ready for user input from the start.

**Comment by AlCalzone** [comment]
Uhh, what are you using to view the readme?
Looks fine to me:
![grafik](https://user-images.githubusercontent.com/17641229/189732186-ca50aff7-d8cd-4e02-952f-7c17cd924def.png)

**Comment by AlCalzone** [comment]
We put the developer information there on purpose. It is the first thing you see when opening a newly generated repo and even newbie devs **will** see it.
From experience, comments and explanations in other files frequently do not get read.

**Comment by mcm1957** [comment]
at B)
I see the intent and you are completly right that the README.md is the best place to ensure something is read.
So please forget point B and keep the current file contents.

at A)
I read the contents within vsCode Editor. I will create a new test adapter, check in detail where I see what info and come back.

**Comment by AlCalzone** [comment]
Feel free to share a repository you created where this happens.

**Comment by mcm1957** [comment]
I'm very sorry. The problem must be caused by some action done by me.

I've created a new test adapter. The readme looks complety ok without any issue and with correct formatting.
Checked by opening it with nodepad++ adn vsCode on wndows.

I've checked further. The malformatted data is store in a README-developer.md which I created to save this information for reference. This file is definitly destroyed. But I only remeber that I did fill this fileusing CTRL-C / CTRL-V copying. I do not remember whetehr I did it with vsCode or with Ultraedit. Anyway A short test with both did NOT reproduce the problem.

So I must hav done some extra action or have pressed an addional key etc. 

****ANYWAY**
There is NO problem with adapter creator. I'll close the issue. Sorry for the addional work load I have caused.** 
The roblem was onec emor in formt of the keyboard.
